BibTeX.js

```diff
@@ -272,6 +272,8 @@ function processField(item, field, value, rawValue) {
 		}
 	} else if(inputFieldMap[field]) {
 		item[inputFieldMap[field]] = value;
+    } else if(field == "location") {
+        item.place = value;
```

Comment:
That is unnecessary, because there is already https://github.com/twsh/translators/blob/9de98ca98b5548c53987ed839f20bd20600ab61f/BibTeX.js#L173